modules/iam-profile/main.tf

```diff
@@ -64,6 +64,7 @@ resource "aws_iam_policy" "ReadOnlyPolicy" {
               "ec2:DescribeAccountAttributes",
               "ec2:GetEbsEncryptionByDefault",
               "ec2:SearchTransitGatewayRoutes",
+              "ec2:GetTransitGatewayMulticastDomainAssociations",
```

Comment:
please keep them in lexicographical manner

Reply:
Fixed.